### ISSUE TITLE
IMDB Plugin no longer working

### ISSUE THREAD

**Issue by palle890** [micjahn/Griffith#8]
Hello Mike,
The current IMDB Plugin lately stopped getting any search results.
I would greatly appreciate if you could look into the problem and update the plugin if possible.
Thanks and regards.

**Comment by micjahn** [comment]
The IMDb page seems to be completely rewritten. Not sure if I will find enough free time to fix the plugin.
Perhaps you should better try this fork of griffith:
https://github.com/Dougis1/Griffith

**Comment by Filbuntu** [comment]
Thank you, Mike, for your quick answer! It is a pity IMDb was rewritten! I still love and use Griffith. I have not found a good alternative. I hope you find time to fix it.
Is the Dougis1-fork any good? How to install in Linux (e.g. Ubuntu)? I tried to find out on the github-page, I could not find an instruction I would understand as a non-programmer.

**Comment by Filbuntu** [comment]
@palle890 Did you find a solution?

**Comment by Filbuntu** [comment]
@micjahn Will you find time to help? Or is https://www.themoviedb.org an alternative? It also has an API and seems more open source.

**Comment by palle890** [comment]
@Filbuntu No, have not found any solution. So far, I always have to manually add all entries in griffith to add new movies which is very unsatisfactory. I have also looked into Dougis1 fork but have not found any solution there because I am a Windows user and there is no ready to use compiled EXE package available and I am not experienced enough to compile a package myself. So I am still hoping that Mike will find some time to help.

**Comment by Filbuntu** [comment]
Let's hope, Michael will find time as Griffith is cool and unique.

https://github.com/aleiepure/ticketbooth/ looks like a promising app but there is no import feature for the Griffith database (yet).
